terraform/environment/region/modules/s3_antivirus/lambda_function.tf

```diff
@@ -18,7 +22,7 @@ resource "aws_lambda_function" "lambda_function" {
   runtime          = "provided.al2023"
   timeout          = 300
   memory_size      = 4096
-  publish          = true
+  publish          = filebase64sha256("${path.module}/myFunction.zip") != terraform_data.replacement.triggers_replace[0] ? true : false
```

Comment:
wouldn't `!=` give you a `true` or `false`?

Reply:
If there's a difference between the current sha and what's stored in the trigger, publish = true, is what' I'm going for.

Reply:
oh I see what you mean, yes, I could leave it as `a != b`